app/src/main/kotlin/com/mapbox/vision/teaser/SignDetectionFragment.kt

```diff
@@ -15,7 +16,11 @@ class SignDetectionFragment : BaseVisionFragment() {
 
     companion object {
         val TAG: String = SignDetectionFragment::class.java.simpleName
-        fun newInstance() = SignDetectionFragment()
+        fun newInstance(country: Country): SignDetectionFragment {
+            val fragment = SignDetectionFragment()
```

Comment:
nit : `apply` seems to fit perfectly here

Reply:
Indeed.